backend/app/usecase/shortlink/updater_test.go

```diff
@@ -19,12 +19,27 @@ func TestShortLinkUpdaterPersist_UpdateShortLink(t *testing.T) {
 
 	now := time.Now().UTC()
 
+	// ShortLink creator and updater methods require string pointers
+	// It is not possible to reference a literal string directly in Go
+	// To make the test suite as clean as possible, they are instead declared below and
+	// the variables are passed into the test case by reference, which is a legal semantic in Go.
+	// https://golang.org/ref/spec#Address_operators
```

Comment:
Let's define 
```
func String(str string) *string {
	return &str
}
```
inside `backend/app/fw/ptr/string.go` and use this function to directly embed string in the test case.